gen/DocumentFormat.OpenXml.Generator.Models/Generators/Elements/DataModelWriterExtensions.cs

```diff
@@ -11,6 +11,112 @@ namespace DocumentFormat.OpenXml.Generator.Generators.Elements;
 
 public static class DataModelWriterExtensions
 {
+    public static class AttributeStrings
+    {
+        public const string ObsoletePropertyWarn = "[Obsolete(\"Unused property, will be removed in a future version.\", false)]";
```

Comment:
Do we need all these? looks like only two are used

Reply:
Not right now, but I added them to make clear the distinction in the different types of obsolescence so we follow some pattern in naming.

Reply:
Not a blocker, but I prefer to compose them from smallerbits of text to ensure they're built up the same.